tide_core.drush.inc

```diff
@@ -0,0 +1,188 @@
+<?php
+
+/**
+ * @file
+ * Drush file to define custom drush command redirect-import.
+ */
+
+use Drupal\redirect\Entity\Redirect;
+
+/**
+ * Implements hook_drush_command().
+ */
+function tide_core_drush_command() {
+  $items = [];
+
+  $items['redirect-import'] = [
+    'description' => "Imports a file and adds redirects to the redirect table",
+    'arguments' => [
+      'filepath' => 'Path to CSV (comma-separated), JSON or PHP file containing old and new URLs for redirects',
+    ],
+  ];
+
+  return $items;
+
+};
+
+/**
+ * Get the node id of the page to redirect to.
+ */
+function get_node_id($to) {
```

Comment:
is this function getting a node id? It seems to be normalising URLs if anything if I understand it correctly.